service/kvmconfig/v10/version_bundle.go

```diff
@@ -10,8 +10,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "component",
-				Description: "Add your change here.",
+				Component:   "cloudconfig",
+				Description: "Improved encryption key injection.",
```

Comment:
Uncertain if we should present that to the customer. Security-wise it doesn't change anything (for kvm-operator).

Reply:
IMO we should track everything in the changelogs. When there will be a requirement to filter changelogs for customers we can add this quite easily.